### ISSUE TITLE
Report Bugs here

### ISSUE THREAD

**Issue by kurokirasama** [iamtalhaasghar/yewtube#7]
Hello, I tried the new pip version, streaming and downloading works great, but when I tried userpl command I got the following error:

```
Error fetching data. Possible network issue.
'NoneType' object has no attribute 'get'
```

## Issue / Suggestion

userpl command doesn't work

## Your Environment

Ubuntu 20.04.3 LTS x86_64

yewtube version    : 1.1.3 released on 08 Feb 2022
yt_dlp version     : 2022.02.04
Python version     : 3.8.10 (default, Nov 26 2021, 20:14:08) 
[GCC 9.3.0]
Processor          : x86_64
Machine type       : x86_64
Architecture       : 64bit, ELF
Platform           : Linux-5.13.0-28-generic-x86_64-with-glibc2.29
sys.stdout.enc     : utf-8
default enc        : utf-8
Config dir         : /home/kira/.config/mps-youtube
env:TERM           : xterm-256color
env:SHELL          : /bin/bash
env:LANG           : en_US.UTF-8

mpv 0.33.0 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on Thu Nov 26 06:59:44 UTC 2020
FFmpeg library versions:
   libavutil       56.60.100
   libavcodec      58.112.103
   libavformat     58.64.100
   libswscale      5.8.100
   libavfilter     7.90.100
   libswresample   3.8.100
FFmpeg version: git-2020-11-22-9208b72

**Comment by iamtalhaasghar** [comment]
For now,  `user <username / query>` `userpl <username>` will work same as `user <username>` because **querying specific videos from a channel** or **getting playlists form the channel** **without Youtube API v3** are not supported yet.
So, just update `yewtube` and don't forget to give your feedbacks / suggestions.

**Comment by VanackSabbadium** [comment]
Is this the same issue when i type

"i numberofvideo"

to get infos about it?

**Comment by iamtalhaasghar** [comment]
@VanackSabbadium yes this feature was also broken when i removed YouTube API but we can get video info and comments without api key so its fixable.

**Comment by iamtalhaasghar** [comment]
closed in favor of #12 #13 #14 so that we can separately track progress of each issue
